src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -113,15 +114,16 @@ const buildCrossReferenceGroups = (crossReferences: CrossReference[]) => {
 const GeneExternalReferences = () => {
   const params: EntityViewerParams = useParams();
 
-  const entityId = params.entityId?.split(':').pop();
+  const { entityId, genomeId } = params;
+
+  const stableId = entityId ? parseEnsObjectIdFromUrl(entityId) : null;
```

Comment:
`parseEnsObjectIdFromUrl` returns an object containing fields `type` and `objectId`. I believe you want the objectId field, not the whole object.

Reply:
Yes, I missed that. Fixed it now.